pages/k8s/using-vault.md

```diff
@@ -16,20 +16,20 @@ toc: True
 As mentioned in the [Certificates and trust reference][certs-doc] documentation,
 [HashiCorp's Vault][vault] can be used to provide either a root or intermediate CA. It can
 also be deployed HA, as well as provide a secure secrets store which can be used to enable
-[encryption-at-rest for **CDK**][encryption-doc].
+[encryption-at-rest for **Charmed Kubernetes**][encryption-doc].
 
 Vault does require an additional database to store its data and (depending on
 configuration) some manual steps will be required after deployment or any reboot so
 that secrets, such as certificates and signing keys, can be accessed.
 
-## Deploying CDK with Vault as a root CA
+## Deploying Charmed Kubernetes with Vault as a root CA
 
-When deploying **CDK** manually via the [published Juju bundle][cdk-bundle], it is
-possible to make use of an overlay file to change the composition and  configuration of
-**CDK**
+When deploying a **Charmed Kubernetes** manually via the
```

Comment:
s/ a //

Reply:
well, there was bound to be at least one of those.